front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -135,7 +135,13 @@ const StdcmConfig = ({
   }, [infra]);
 
   useEffect(() => {
-    dispatch(resetStdcmConfig());
+    const keepForm = localStorage.getItem('keepForm');
+
+    if (!keepForm) {
+      dispatch(resetStdcmConfig());
+    } else {
+      localStorage.removeItem('keepForm');
```

Comment:
```suggestion
      localStorage.removeItem('keepForm');
      localStorage.removeItem('inputsData');
```
Shouldn't we clear stored inputs data to prevent stale or unnecessary data from persisting across sessions or tabs ?

Reply:
The ```inputData```const was removed because it was actually unnecessary.